### PR TITLE
Bugfix/TEC-808 Change 'statues' to 'statuses'

### PR DESCRIPTION
**Pull request by Dollar0712**
Changed spelling error of "statuses"

### This will only work after the spelling error in the frontend is also fixed

### CODE REVIEW

**Review by oceanicshadow88 — APPROVED** [review]

**Review by oceanicshadow88 — CHANGES_REQUESTED** [review]
test

**Comment by Dollar0712** [comment]
Added test for get project details and passed.


This change affects:
- Get project details endpoint: get /projects/:id/details
- Create project: post /projects

### Note: First 2 tests are not passed based on develop branch
All **related** tests passed.

File: test/__tests__/project.test.js [hunk 1]
```diff
@@ -57,4 +57,21 @@ describe('Project Test', () => {
       .send({ ...project, ...{ key: null } })
       .expect(422);
   });
+
+  it('should get project details', async () => {
+    const project = await new ProjectBuilder().save();
+    const res = await request(app.application).get(`/api/v2/projects/${project.id}/details`);
+
+    expect(res.statusCode).toBe(200);
+    expect(res.body).toHaveProperty('labels');
```
Comment (xuewei-li-L) [inline test/__tests__/project.test.js:66]: ProjectBuilder doesn't automatically create labels, users, or ticketTypes, and the API response returns empty values for these fields, would it make more sense for our test data to remain empty as well?

**Review by xuewei-li-L — DISMISSED** [review]

File: test/__tests__/project.test.js (hunk 1 above)
Comment (oceanicshadow88) [reply test/__tests__/project.test.js:66]: If your test only checks expect(res.body).toHaveProperty('users'), it's essentially no better than having no test—you're not validating anything meaningful

**Comment by Dollar0712** [comment]
Changed test to match the actual value & passed the relative test
image

**Review by oceanicshadow88 — APPROVED** [review]
